applications/nrf_desktop/src/modules/Kconfig.hid

```
@@ -37,6 +37,13 @@ config DESKTOP_HID_DONGLE
 
 menu "HID Reports"
 
+config DESKTOP_HAS_HID_OUTPUT_REPORT
+	bool
+	help
+	  This option indicates that the device supports HID output reports.
+	  It must by selected to handle receiving HID output reports. By default
```

`by` --> `be`, and as this setting is promptless we should probably inform the reader of this text that it must be selected from another Kconfig setting.

```suggestion
	  It must be selected from other Kconfig setting to handle receiving HID output reports. By default
```

---

Fixed